site/src/Entity/Notification.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Mgilet\NotificationBundle\Entity\NotificationInterface;
+use Mgilet\NotificationBundle\Model\Notification as NotificationModel;
+
+/**
+ * A Notification to be sent
+ *
+ * Overrides Mgilet\NotificationBundle\Entity\Notification
+ *
+ * @ORM\Table("Notification")
+ * @ORM\Entity
+ */
+class Notification extends NotificationModel implements NotificationInterface
+{
+    /**
+     * The recipient of the Notification
+     *
+     * @var User
+     *
+     * @ORM\Column(name="recipient", nullable=false)
+     *
+     */
+    protected $recipient;
```

Comment:
You don't need that, the recipient is in the notifiable annotation in mgilet-notificationbundle.

Reply:
How can we fetch that while persisting the Notification for a specific user ?

Reply:
Cause not having a recipient while persisting crashes our consumer

Reply:
You added an annotation to make the User entity a NotifableEntity; that's where the recipient of a message is stored in this case.

Reply:
That is correct, what was I even thinking!